_config.yml

```diff
@@ -5,7 +5,9 @@ baseurl: /plugins
 collections:
   plugins:
     output: true
-    permalink: /:title/
+    permalink: /:title/ # DO NOT CHANGE THIS DEFAULT
```

Comment:
How do you feel about changing this to `/plugins/:title/`? This would allow us to differentiate from (maybe) `/themes/`

Of course, themes and plugins should probably not share names, so maybe this is no big deal.